py-deepstream-text-detection/README.md

```diff
@@ -41,10 +49,32 @@ optional arguments:
                         The approach to do objec tracking, default=NvDCF
 ```
 
-# Run
+## Prepare environments
+
+`pip3 install -r requirements.txt`

```

Comment:
do you mean requirements from the parent folder?

Reply:
Yes, I changed to `../requirements.txt`